src/Core/Addon/Module/ModuleManager.php

```diff
@@ -71,15 +73,24 @@ class ModuleManager implements AddonManagerInterface
      */
     private $translator;
 
+    /**
+     * @var Employee Legacy employee class
+     */
     private $employee;
+    
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $dispatcher;
 
     public function __construct(AdminModuleDataProvider $adminModulesProvider,
         ModuleDataProvider $modulesProvider,
         ModuleDataUpdater $modulesUpdater,
         ModuleRepository $moduleRepository,
         ModuleZipManager $moduleZipManager,
         TranslatorInterface $translator,
-        Employee $employee = null)
+        Employee $employee = null,
+        EventDispatcherInterface $dispatcher = null)
```

Comment:
You can create a ``NullDispatcher`` that implement the interface and check that events are dispatched or simply [mock the interface](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Tests/FormConfigTest.php#L74)

The real issue is still to rely on same class on both legacy and symfony parts :/